openerp/addons/base/ir/ir_ui_view.py

```diff
@@ -1193,15 +1194,29 @@ def _validate_custom_views(self, cr, uid, model):
         cr.execute("""SELECT max(v.id)
                         FROM ir_ui_view v
                    LEFT JOIN ir_model_data md ON (md.model = 'ir.ui.view' AND md.res_id = v.id)
-                       WHERE md.module NOT IN (SELECT name FROM ir_module_module)
+                       WHERE (md.module NOT IN (SELECT name FROM ir_module_module)
+                              OR md.module IS NULL)
```

Comment:
The comment upstream is that you can put this with `IS TRUE`: https://github.com/odoo/odoo/pull/23626#issuecomment-372977175